adafruit_binascii.py

```diff
@@ -0,0 +1,190 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2014 Paul Sokolovsky
+# Modified by Brent Rubell for Adafruit Industries, 2019
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`adafruit_binascii`
+================================================================================
+
+Helpers for conversions between binary and ASCII
+
+
+* Author(s): Paul Sokolovsky, Brent Rubell
+
+Implementation Notes
+--------------------
+
+**Hardware:**
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+try:
+    from binascii import hexlify, unhexlify
+except ImportError:
+    pass
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_binascii.git"
+
+
+TABLE_A2B_B64 = [
```

Comment:
unless this is modified, please make it a () instead of []

Reply:
or better yet a b'' bytearrays (most compact)

Reply:
@ladyada  Made this a tuple instead of a list, unsure how to handle the negative values within `TABLE_A2B_B64` if it were changed to a bytearray.